src/utils/serialize.ts

```diff
@@ -9,14 +9,14 @@ const getUserCourseSessionDetails = (userObject: UserEntity) => {
       name: course.course.name,
       slug: course.course.slug,
       status: course.course.status,
-      storyblokId: course.course.storyblokId,
+      storyblokId: Number(course.course.storyblokId),
```

Comment:
why are we converting this to a number?

Reply:
While working with Storyblok, I realised story id's are numbers

Reply:
Their type is number - Typescript wont know it's a number until I wrap it in Number()